bibtex/bib/quantum.bib

```diff
@@ -51,6 +51,19 @@ @Article{AFG+12
   Url                      = {http://doi.acm.org/10.1145/2240236.2240258}
 }
 
+@InProceedings{AMTW00,
+  Title                    = {Private quantum channels},
+  Author                   = {Ambainis, Andris and Mosca, Michele and Tapp, Alain and de Wolf, Ronald},
+  Booktitle                = {Foundations of Computer Science, 2000. Proceedings. 41st Annual Symposium on},
```

Comment:
I was trying to consistently use: 
41st Annual Symposium on Foundations of Computer Science - FOCS 2000  for FOCS conferences